frontend/app/src/py-handler.ts

```diff
@@ -228,7 +228,7 @@ export default class PyHandler {
       client.on('error', e => {
         reject(e);
       });
-      client.kill();
+      client.kill('SIGINT');
```

Comment:
This we discussed in the standup. Is it needed for this change or is it remnant of your work trying to fix https://github.com/rotki/rotki/issues/2808 ?

Reply:
This is indeed a leftover